epyqlib/tabs/files/filesview.py

```diff
@@ -37,10 +39,25 @@ def setup_ui(self):
         self.ui.setupUi(self)
         print("[Filesview] setup_ui called")
         self.ui.something.clicked.connect(self.echo)
+        self.ui.get_serial_number.clicked.connect(
+            lambda: self.get_serial_number(),
```

Comment:
There are umpteen options doing this connection.  Every option I've come up with has aspect I dislike so...  here's one.